src/DocsGenerator/Helpers/CategoryHelper.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using Codacy.Engine.Seed.Patterns;
+
+namespace Codacy.TSQLLint.DocsGenerator.Helpers
+{
+    /// <summary>
+    /// Helper to convert to codacy category from a TSQLLint rule
+    /// </summary>
+    public static class CategoryHelper
+    {
+        /// <summary>
+        /// This map a rule to a codacy category.
+        /// </summary>
+        /// <param name="rule">rule name</param>
+        /// <returns>Codacy category</returns>
+        /// <exception cref="NotImplementedException">When found a non explicit mapped rule</exception>
+        public static Category ToCategory(string rule)
+        {
+            switch (rule)
+            {
+                case "data-compression":

```

Comment:
shouldn't this come from a configuration file instead to be hard-coded?

Reply:
This is to map the rule to our `Category`, and the configuration file doesn't have it. On this tool, unfortunately, we need to do it manually.